helpers/test_schema.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "result": {
+      "type": "string",
+      "enum": ["SUCCESS", "FAILURE", "WARNING", "SKIPPED", "ERROR"]
+    },
+    "namespace": {
+      "type": "string"
+    },
+    "timestamp": {
+      "type": "string",
+      "pattern": "^[0-9]{10}$"
+    },
+    "successes": {
+      "type": "integer",
+      "minimum": 0
+    },
+    "note": {
+      "type": "string"
```

Comment:
I think we do need the `maxLength` set here, something like 512 could be a good limit?

Reply:
`maxLength` for `note` was updated to be 512.